From 04e3ae81d79b6304d1a075480986a567a4d8f1a7 Mon Sep 17 00:00:00 2001
From: isch4196 <isch4196@colorado.edu>
Date: Thu, 1 Jun 2023 10:25:27 -0700
Subject: [PATCH] Changed insmod to modprobe

---
 misc-modules/module_load | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/misc-modules/module_load b/misc-modules/module_load
index a669c5e..b107ed8 100755
--- a/misc-modules/module_load
+++ b/misc-modules/module_load
@@ -22,7 +22,7 @@ else
 fi
 
 echo "Load our module, exit on failure"
-insmod ./$module.ko $* || exit 1
+modprobe ./$module.ko $* || exit 1
 echo "Get the major number (allocated with allocate_chrdev_region) from /proc/devices"
 major=$(awk "\$2==\"$module\" {print \$1}" /proc/devices)
 if [ ! -z ${major} ]; then
